src/logic/safe/utils/modules.ts

```diff
@@ -64,3 +65,7 @@ export const enableModuleTx = ({
     notifiedTransaction: TX_NOTIFICATION_TYPES.SETTINGS_CHANGE_TX,
   }
 }
+
+export const isModuleEnabled = (modules: string[] | undefined | null, moduleAddress: string): boolean => {
```

Comment:
why the type here is so broad? I believe this function works correctly only if `string[]` is passed